server/api_channel.go

```diff
@@ -103,6 +103,13 @@ func (p *Plugin) handleSince(c *gin.Context) {
 		return
 	}
 
+	p.track(evSummarizeUnreadMessages, map[string]any{
```

Comment:
Can we have it be `evUnreadMessages` since it does more than summary?

Reply:
Done